app/views/companies/show.html.haml

```diff
@@ -1,92 +1,101 @@
-    .row
-      .col-md-5
-        = field_or_none t('.name'),@company.name, tag_options: {class: 'hidden', id: 'location-text'}
+  - content_for :head_content do
+    = Adapters::CompanyToSchemaLocalBusiness.new(@company, url: company_url(@company)).as_target.to_ld_json.html_safe
 
-        - if current_user.admin?
-          = field_or_none t('.org_nr'), @company.company_number, tag_options: {class: 'company-number'}
 
-        = field_or_none t('.phone_number'), @company.phone_number, tag_options: {class: 'phone-number'}
+  %div{ id: "#{unique_css_id(@company)}",
+        class: "#{item_view_class(@company, controller.action_name)}" }
 
-        = field_or_none t('.email'), mail_to(@company.email), tag_options: {class: 'email'}
+    %header.entry-header
+      %h1.entry-title= @company.name
+    .entry-content
+      .row
+        .col
+          - if @company.categories
+            .categories
+              = render 'business_categories/as_list',
+                categories: @company.business_categories
+          - if payment_visible_for_user?(current_user, @company)
+            %hr
+            = render partial: 'branding_payment_status', locals: { company: @company }
+            %hr
 
-        = field_or_none t('.website'), link_to(@company.website, sanitize(full_uri(@company)), target: '_blank'), tag_options: {class: 'website'}
+      .row
+        .col.company-story
+          != @company.description
+        %hr
 
-      .col-md-7
-        - if @company.any_visible_addresses?
-          = render partial: 'map_companies',
-                   locals: { markers: location_and_markers_for([@company],
-                   link_name: false )}
+      .row
+        .col-md-5
+          = field_or_none t('.name'),@company.name, tag_options: {class: 'hidden', id: 'location-text'}
 
-    - user_can_edit = policy(@company).update?
+          - if current_user.admin?
+            = field_or_none t('.org_nr'), @company.company_number, tag_options: {class: 'company-number'}
 
-    = render 'company_members', company: @company
+          = field_or_none t('.phone_number'), @company.phone_number, tag_options: {class: 'phone-number'}
 
-    = render 'company_addresses', company: @company, user_can_edit: user_can_edit
+          = field_or_none t('.email'), mail_to(@company.email), tag_options: {class: 'email'}
 
-    - if user_can_edit
+          = field_or_none t('.website'), link_to(@company.website, sanitize(full_uri(@company)), target: '_blank'), tag_options: {class: 'website'}
 
-      = link_to "#{t('.add_address')}",
-                new_company_address_path(@company),
-                class: 'btn btn-info btn-sm ml-2 add-company-address'
+          .social-media
+            = icon_link(@company.facebook_url, "facebook") if !@company.facebook_url.blank?
+            = icon_link(@company.instagram_url, "instagram") if !@company.instagram_url.blank?
+            = icon_link(@company.youtube_url, "youtube-square") if !@company.youtube_url.blank?
```

Comment:
These might be cleaner as:
```
= icon_link(@company.facebook_url, "facebook") if @company.facebook_url.present?
```
(a matter or preference - but it's one less method call and avoids a "double negative")

Reply:
That's definitely much, much better.  (And really I should have used `unless` instead of `if !...`)  Patrick's suggestion is best.